src/platform/site-wide/user-nav/containers/AutoSSO.jsx

```diff
@@ -28,7 +28,7 @@ function AutoSSO(props) {
     removeLoginAttempted();
   }
 
-  const badPaths = ['auth/login/callback', 'logout', 'terms-of-use'];
+  const badPaths = ['auth/login/callback', 'logout'];
```

Comment:
Yes, so we originally had blocked auto sso on the terms of use page because we didn't want the user to automatically be signed in without grabbing where they were redirecting to.

Also, I think Trevor knows how to test it somehow